agent/containers/images/image-test-script.sh

```diff
@@ -0,0 +1,283 @@
+    fi
+done
+printf -- "\nDone selecting hosts:\n"
+cat ${pbench_run}/remotes.lis
+printf -- "\nNow to select tools.\n\n"
+
+# Register the specified tools, will be recorded in
+# ${pbench_run}/tools-v1-default
+while :; do
+    read -p "Tool name (type 'help' to see options): " tool
+    if [ -z "$tool" ]
+    then
+        printf -- "Tool name empty. Please write a tool name.\n"
+    else
+        if [ $tool == "help" ]
+        then
+            printf -- "\nAvailable tools:\n"
+            python3 -c "import sys, json; meta = json.load(open(sys.argv[1])); print('  Transient:', *[f'\t{tool}' for tool in meta['transient'].keys()], '  Persistent:', *[f'\t{tool}' for tool in meta['persistent'].keys()], sep='\n')" /opt/pbench-agent/tool-scripts/meta.json
+        else
+            cmd="pbench-register-tool --name=${tool} --remotes=@${pbench_run}/remotes.lis"
+            printf -- "\n$cmd\n"
+            $cmd
+            read -p "Another tool? (y/n)" anothertool
+            if [ $anothertool != "y" ]
+            then
+                break
+            fi
+        fi
+    fi
+done
+printf -- "\nDone registering tools.\n\n"
+ls -lR ${pbench_run}/tools-v1-default
+
+printf -- "\n\n"
+wait_keypress 120
+
+
+# Start Tool Meister containers on remote host
+
+printf -- "\n\nNow please start the Tool Meister container on the specified host...\n\n\t$ podman run --name pbench-agent-tool-meister \\ \n\t\t--network host --ulimit nofile=65536:65536 --rm -d \\ \n\t\t-e REDIS_HOST=${REDIS_HOST} \\ \n\t\t-e REDIS_PORT=${REDIS_PORT} \\ \n\t\t-e PARAM_KEY=tm-default-\$(hostname -f) \\ \n\t\t-e _PBENCH_TOOL_MEISTER_LOG_LEVEL=debug \\ \n\t\tquay.io/pbench/pbench-agent-tool-meister-${DISTRO}:${TAG}\n\n"
+
+wait_keypress 120
+
+
+# Start the Tool Data Sink locally, mapping our volume into the Tool Data Sink
+# container.
+
+printf -- "\n\nNow we will automatically start the local Tool Data Sink to pull tool data into our local volume.\n\n"
+
+set -x
+podman run --name pbench-agent-tool-data-sink --network host --volume ${pbench_run}:/var/lib/pbench-agent:Z --ulimit nofile=65536:65536 --rm -d -e REDIS_HOST=${REDIS_HOST} -e REDIS_PORT=${REDIS_PORT} -e PARAM_KEY=tds-default -e _PBENCH_TOOL_DATA_SINK_LOG_LEVEL=debug quay.io/pbench/pbench-agent-tool-data-sink-${DISTRO}:${TAG}
+set +x
+
+printf -- "\n\nOptional: Now let's look at the 'podman logs' output from these containers;\n\t You will notice if we did not start a Redis server yet,\n\tthey'll just be waiting for it to show up.\n\n"
+
+wait_keypress 120
+
+
+printf -- "\n\nPlease start the Redis server on ${REDIS_HOST}:${REDIS_PORT} if not currently up\n\t$ podman run --name demo-tm-redis -p ${REDIS_PORT}:6379 --rm -d redis\n\n\tNote TDS and TMs notice Redis server,\n\tbut now wait for their 'PARAM_KEY' to show up.\n\n"
+
+wait_keypress 120
+
+export date="2021-1-1T12:00:00"
+
+source /etc/profile.d/pbench-agent.sh
+source /opt/pbench-agent/base
```

Comment:
Why do we set `date` to a fixed value instead of letting `/opt/pbench-agent/base` set it to the current value?  (This would be worth an explanatory comment.)